recipes/reparation/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "reparation" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "e3438e35d5b32de05d7fa8928c83647b3ab644df4acbdaf75c0ccc5ef2c5b2d4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/biobix/{{name}}/archive/259d113f838136154c3e82b2ed3518027724332e.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - positive_set.patch
+
+build:
+  number: 0
+  skip: True [osx or py35]
+
+requirements:
+  host:
+    - perl
+    - perl-posix
```

Comment:
you do not need all the build requirements isn't it?

Reply:
Yes thanks, my bad.